### PR TITLE
Chore/neo blog storybook

### PR DESCRIPTION
**Pull request by mahdisolgi1**
### PR Summary:
This pull request includes a few of updates and enhancements to various components within the project.

#### General Improvements:

- Removed unnecessary stackdump.exe file.

- Installed and set up Storybook for UI component development.

- Added the Button component to Storybook for easier visual testing.

- Added a README file with a project description and setup instructions.

### CODE REVIEW

File: frontend/src/stories/Button.stories.ts
Comment (amirrezaDev1378) [inline frontend/src/stories/Button.stories.ts:1]: remove comments and rename the title

File: frontend/src/stories/Configure.mdx
Comment (amirrezaDev1378) [inline frontend/src/stories/Configure.mdx:1]: remove this

File: frontend/src/stories/Header.stories.ts
Comment (amirrezaDev1378) [inline frontend/src/stories/Header.stories.ts:1]: remove comment and rename title

File: frontend/src/stories/Page.tsx
Comment (amirrezaDev1378) [inline frontend/src/stories/Page.tsx:1]: remove this

File: frontend/src/stories/assets/accessibility.png
Comment (amirrezaDev1378) [inline frontend/src/stories/assets/accessibility.png:1]: remove

File: frontend/src/stories/assets/accessibility.svg
Comment (amirrezaDev1378) [inline frontend/src/stories/assets/accessibility.svg:1]: remove all unused asstes ...

File: package.json
Comment (amirrezaDev1378) [inline package.json:1]: why did you add this ? either remove it or merge it

**Review by amirrezaDev1378 — CHANGES_REQUESTED** [review]

**Comment by mahdisolgi1** [comment]
I removed stories folder because it wasn't used

**Review by amirrezaDev1378 — APPROVED** [review]

**Pull request merged**